assets/l10n/app_en.arb

```diff
@@ -49,6 +333,42 @@
       }
     }
   },
+  "serverExceptionMalformedResponse": "Server gave malformed response; HTTP status {httpStatus}",
+  "@serverExceptionMalformedResponse": {
+    "description": "Message for error dialog when an API call fails because we could not parse it.",
+    "placeholders": {
+      "httpStatus": {
+        "type": "int",
+        "example": "500"
```

Comment:
nit: this is unrepresentative because for HTTP status 500 we'll never use this message (see the `MalformedServerResponseException` constructor call sites, and the `httpStatus` logic in `_makeApiException`). Probably most typical is 200:
```suggestion
  "serverExceptionMalformedResponse": "Server gave malformed response; HTTP status {httpStatus}",
  "@serverExceptionMalformedResponse": {
    "description": "Message for error dialog when an API call fails because we could not parse it.",
    "placeholders": {
      "httpStatus": {
        "type": "int",
        "example": "200"
```